rubix-spi/src/main/java/com/qubole/rubix/spi/fop/ObjectPoolPartition.java

```diff
@@ -144,8 +146,8 @@ public void returnObject(Poolable<T> object)
         log.warn(String.format("Could not increase pool size. Pool state: totalCount=%d queueSize=%d delta=%d", totalCount, objectQueue.size(), delta));
```

Comment:
name left out in the log line. Better have a common log function in this class which adds `name` and use it for all logging here.

Reply:
added custom logger for ObjectPoolPartition.